### PR TITLE
feat(Task-13): As a User, I want to see a detailed report for a day

### PR DESCRIPTION
**Pull request by jLagosCarrera**

### CODE REVIEW

File: src/app/shared/components/contact-form/contactForm.controller.js [hunk 1]
```diff
@@ -1,7 +1,15 @@
 export default class ContactFormController {
-    constructor($mdDialog, $scope) {
+    constructor($mdDialog, $scope, $rootScope) {
         this.$mdDialog = $mdDialog;
         this.$scope = $scope;
+        this.$rootScope = $rootScope;
+
+        this.$rootScope.$on('$locationChangeStart', ($event) => {
```
Comment (eloydotco) [inline src/app/shared/components/contact-form/contactForm.controller.js]: unsubscribe this listener

File: src/app/shared/components/detailed-day/detailedDay.controller.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+export default class DetailedDayController {
+    constructor($mdDialog, $scope, $rootScope) {
+        this.$mdDialog = $mdDialog;
+        this.$scope = $scope;
+        this.$rootScope = $rootScope;
+
+        this.$rootScope.$on('$locationChangeStart', ($event) => {
```
Comment (eloydotco) [inline src/app/shared/components/detailed-day/detailedDay.controller.js]: unsubscribe this listenner

File: src/app/shared/components/forecast-card/forecastCard.controller.js [hunk 3]
```diff
@@ -1,13 +1,30 @@
...
     }
 
     getRotation() {
         if (this.currentForecast) {
             return `rotate(${this.currentForecast.wind.deg}deg)`;
         }
     }
+
+    openDialog(day) {
+        const dialogScope = angular.extend(this.$scope.$new(true), {
```
Comment (eloydotco) [inline src/app/shared/components/forecast-card/forecastCard.controller.js:15]: check if this scope is destroyed with the dialog, in negative case, destroy it when dialog is closed

**Review by eloydotco — CHANGES_REQUESTED** [review]

File: src/app/shared/components/detailed-day/detailedDay.component.scss [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+            padding: 0;
+        }
+
+        md-list-item {
+            background-color: rgba(229, 229, 229, 0.6);
+            border-radius: 5px;
+            margin: 4px 4px 0px 4px;
+
+            &:last-of-type {
+                margin: 4px 4px 4px 4px;
```
Comment (gbtec-ext-jorgevillar) [inline src/app/shared/components/detailed-day/detailedDay.component.scss]: A simple `maring: 4px;` would be enought ;)

**Review by gbtec-ext-jorgevillar — APPROVED** [review]

**Review by eloydotco — APPROVED** [review]

**Pull request merged**